src/main/java/com/training/library/service/DatabaseService.java

```diff
@@ -0,0 +1,95 @@
+package com.training.library.service;
+
+import com.training.library.entity.Book;
+import java.lang.reflect.InvocationTargetException;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class DatabaseService {
```

Comment:
As we discussed this class should be renamed in accordance with DAO or Repository patterns and package as well.